crates/node-bindings/src/core/project_path.rs

```diff
@@ -0,0 +1,76 @@
+use std::path::Path;
+use std::path::PathBuf;
+
+use napi::bindgen_prelude::FromNapiValue;
+use napi::bindgen_prelude::ToNapiValue;
+use napi::sys::napi_env;
+use napi::sys::napi_value;
+
+/// Similar to opaque type ProjectPath in Rust.
```

Comment:
This file demonstrates how to new-type wrap values if needed.

Generally I'd avoid doing this.